internal/state/store.go

```diff
@@ -131,12 +139,26 @@ func IsEmpty(store dbStore) (bool, error) {
 
 // NewStore creates the dbStore of the state pkg.
 func NewStore(db dbm.DB, options StoreOptions) Store {
-	return dbStore{db, options}
+	return &dbStore{
+		db:              db,
+		valInfoCache:    make(map[int64]*cmtstate.ValidatorsInfo),
```

Comment:
Why do we need these? I don't see them used anywhere in this PR

Reply:
I believe that I made a branch management mistake here.  Thanks for review.

Reply:
Hey, I'm wrong.  Basically one finding was that it was faster to use pointers and lay things out first.  That's why that's there.